Euroleague-Graduate/Euroleague-Graduate/TurkishAirLinesHelper.swift

```diff
@@ -0,0 +1,153 @@
+//
+//  TurkishAirLinesHelper.swift
+//  Euroleague-Graduate
+//
+//  Created by Modeso on 7/19/17.
+//  Copyright © 2017 Modeso. All rights reserved.
+//
+
+import Foundation
+import SWXMLHash
+import RealmSwift
+
+protocol TurkishAirlinesHelperDelegate {
+    func updateData(_ table: Results<GameData>) -> Void
+}
+
+class TurkishAirLinesHelper {
+    
+    enum GameDataStatus {
+        case schedule
+        case results
+    }
+    
+    private var turkishViewModelDelegate: TurkishLeagueViewModel? = nil
+    
+    private var games: Dictionary<Int, GameData> = [:]
+    
+    let urls: Dictionary<GameDataStatus, String> = [
+        .schedule : "http://www.euroleague.net/euroleague/api/schedules?seasoncode=E2016",
+        .results : "http://www.euroleague.net/euroleague/api/results?seasoncode=E2016"
+    ]
+    
+    func setMyDelegate(turkishViewModel: TurkishLeagueViewModel){
```

Comment:
use  func setMyDelegate(delegate: TurkishAirlinesHelperDelegate)
because the view model is delegate for this helper class, and already conforms to this protocol... and the helper should't know that such view model exists, ut just knows that there is delegate that needs the updateData( .. Method.